File: v4/nodejs/audio_stream/main.js
const utils = require('../lib/utils')

const url = 'http://api-audiostream-bj.fengkongcloud.com/audiostream/v4'
const access_key = '{ACCESS_KEY}'
const stream_url = '{URL}'
const bt_id = '{BT_ID}'
const uid = '{UID}'

const payload = {
    'accessKey': access_key,
    'appId': 'default',
    'eventId': 'audio',
    'type': 'DEFAULT',
    'btId': bt_id,
    'callback': 'https://jsonplaceholder.typicode.com/posts/',
    'data': {
        'url': stream_url,
        'tokenId': uid,
    }
}

utils.sendRequest(url, 'POST', payload, data => {
    // 通过 code 和 riskLevel 判断返回，具体请参考接口文档
    console.log(data)
})

File: v4/go/audio_stream/main.go
package main

import (
	"bytes"
	"encoding/json"
	"fmt"
	"io/ioutil"
	"net/http"
)

func main() {
	url := "http://api-audiostream-sh.fengkongcloud.com/audiostream/v4"
	accessKey := "{ACCESS_KEY}"
	streamUrl := "{URL}"
	btId := "{BT_ID}"
	uid := "{UID}"

	payload := map[string]interface{}{
		"accessKey": accessKey,
		"appId":     "default",
		"eventId":   "audio",
		"type":      "POLITY_EROTIC_MOAN",
		"callback":  "https://jsonplaceholder.typicode.com/posts/",
		"data": map[string]interface{}{
			"url":     streamUrl,
			"tokenId": uid,
			"btId":    btId,
		},
	}
	b, _ := json.Marshal(payload)
	resp, _ := http.Post(url, "application/json", bytes.NewBuffer(b))

	var data map[string]interface{}
	if resp != nil {
		respBytes, _ := ioutil.ReadAll(resp.Body)
		_ = json.Unmarshal(respBytes, &data)
		// 通过 code 和 riskLevel 判断返回，具体请参考接口文档
		fmt.Println(data)
	}
}
